### PR TITLE
UCP/WORKER: Optimize wakeup arming function.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3992/ for details (Mellanox internal link).

File: src/ucp/core/ucp_worker.c [hunk 1]
```diff
@@ -404,6 +406,8 @@ ucp_worker_add_iface(ucp_worker_h worker, ucp_rsc_index_t tl_id,
         if (status != UCS_OK) {
             goto out_close_iface;
         }
+
+        worker->arm_ifaces_mask |= UCS_BIT(tl_id);
```
Comment (shamisp) [inline src/ucp/core/ucp_worker.c]: where do we check that we don't overrun 64bit ?
Alternatively you can do %

Comment (yosefe) [inline src/ucp/core/ucp_worker.c]: you are right, i don't
will change this to a list instead of bitmap to support larger values of tl_id

Comment (shamisp) [inline src/ucp/core/ucp_worker.c]: I think you can just do the module and cache the tl_id  for final compare.

**Comment by yosefe** [comment]
will be reimplemented

**Pull request closed without merging**